yaml/xyz/openbmc_project/Provisioning/Provisioning.interface.yaml

```diff
@@ -0,0 +1,66 @@
+description: >
+    Interface representing the provisioning state and peer connectivity status
+    of the BMC. It provides properties to indicate whether the BMC has been
+    provisioned and the current connection state with the peer BMC, along with a
+    method to initiate peer provisioning and a signal to report its result.
+
+properties:
+    - name: Provisioned
+      type: boolean
+      flags:
+          - readonly
+      description: >
+          True means the BMC is in a provisioned state.
+    - name: PeerConnected
+      type: enum[self.PeerConnectionStatus]
```

Comment:
Suggestion: If the proposed ProvisonPeer API is intended to support specifying which peer BMC needs to be provisioned, we should also indicate which peer BMC’s connection status it represents. However, since the IBM use case involves only one peer BMC, we can revisit this if we decide to extend support for multiple peers in the future.

That said, we might face challenges on the passive BMC side, as it won’t have a clear way to identify the peer BMC’s position or other distinguishing information to retrieve the corresponding connection status since we know only the local BMC’s position on both sides, and not the peer BMC’s position, due to limitations in the VPD.